vault/resource_gcp_auth_backend_role.go

```diff
@@ -36,11 +36,6 @@ func gcpAuthBackendRoleResource() *schema.Resource {
 			Optional: true,
 			ForceNew: true,
 		},
-		"project_id": {
```

Comment:
Why is this disappearing?

Reply:
It was previously marked as Removed, but was not properly removed in:

https://github.com/hashicorp/terraform-provider-vault/pull/1175/files#diff-733d5d028c1c84512a7399c45cae4cb5a1c7cdddaf281c4a3def2100b7761111L43